standard/template/sections/clause_9_zarr_encoding_overviews.adoc

```diff
@@ -30,18 +49,73 @@ Multiscale metadata SHALL be defined using a `multiscales` attribute located in
 - `resampling_method` – One of the standard string values (e.g., `"nearest"`, `"average"`)
 - `tile_matrix_set_limits` – (optional) Zoom-level limits following the STAC Tiled Asset style
 
-===== Zarr v2 Encoding Example (`.zattrs`)
+The multiscales metadata enables complete discovery of the multiscale collection structure:
+- The TileMatrixSet definition (whether referenced by identifier or included inline) specifies 
+  the exact set of zoom levels through its tileMatrices array
+- Each TileMatrix.id value corresponds to a required child group in the multiscale hierarchy
+- Variable discovery within each zoom level group follows standard Zarr metadata conventions
```

Comment:
The data model part should describe the abstract information (TMS 2.0), and the Zarr encoding should be limited to how/where this model is encoded.

For example:
- data model explains that a "dataset" with overviews should include a multiscale attributes specified with TMS 2.0
- Zarr ecnoding explains the attribute is specified in JSON (and provide optionally the examples).